uaclient/files/files.py

```diff
@@ -42,9 +42,10 @@ def write(self, content: str):
             else defaults.WORLD_READABLE_MODE
         )
         if not os.path.exists(self._directory):
-            os.makedirs(self._directory)
             if os.path.basename(self._directory) == defaults.PRIVATE_SUBDIR:
-                os.chmod(self._directory, 0o700)
+                os.makedirs(self._directory)
```

Comment:
I'd say this is inverted, right?
We only want the mode to be applied if it is the private subdir

Reply:
Good catch, pushing a fix in a few seconds. Since this is no new logical commit I'll squash that fixup with the commit I had and force push.